sympy/core/power.py

```diff
@@ -453,42 +467,44 @@ def _eval_is_positive(self):
             if self.exp.is_imaginary:
                 return log(self.base).is_imaginary
 
-    def _eval_is_negative(self):
-        if self.base.is_negative:
-            if self.exp.is_odd:
+    def _eval_is_extended_negative(self):
+        if self.base.is_extended_negative:
+            if self.exp.is_odd and self.base.is_finite:
```

Comment:
The exponent is odd but the base can still be infinite. This is for e.g. `oo**-1`

Reply:
Current master fails for this case:
```py
In [33]: p = symbols('p', negative=True)

In [34]: q = symbols('q', odd=True)

In [35]: (p**q).is_negative 
Out[35]: True # Shoud be None since p can be q can be negative as well
```

Reply:
This is actually a good example of why it's good to make positive -> finite. Without that, we cannot make a useful deduction here, because the possibility of infinity breaks the fact. Of course, it's a restricted assumption, so anything that is true of extended_positive is also true of positive, but not the other way around. So the end result is fewer `None`s from infinity technicalities. Fewer bugs too, as this shows, because people forget to consider infinity all the time.

Reply:
Before this PR in #16433 I tried to fix the bugs in Pow.is_zero etc without changing the definition of any of the assumptions. If you look there you can see how many parts of the code base I had to change because they were depending on bugs in the assumptions system.